sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -66,14 +70,13 @@ export default defineComponent({
     cancel: null,
   },
   props: {
-    data: {
-      type: Object as PropType<OfferingFormModel>,
+    offeringId: {
+      type: Number,
       required: true,
-      default: {} as OfferingFormModel,
     },
     formMode: {
       type: String as PropType<OfferingFormModes>,
-      required: true,
+      required: false,
```

Comment:
Why was this and line 89 changed, `default` value is supposed to take care of everything when nothing is passed right? Let me know if I am wrong

Reply:
Do you mean `required` that I set to `false`? I saw there are multiple places where the component `offering-form` is called from and `formMode` isn't passed as a prop. Please let me know if I am missing something.

Reply:
Reverted it back as per the discussion.